packages/node_modules/@webex/webex-core/test/integration/spec/services/services.js

```diff
@@ -505,6 +506,23 @@ describe('webex-core', () => {
         services.updateServices();
       });
 
+      it('updates query.email to be emailhash-ed using SHA256', () => {
+        services._fetchNewServiceHostmap = sinon.stub().resolves();
+
+        services.updateServices({
```

Comment:
This needs to either be `return`ed or utilizing the `done` function to make sure the promise finishes within the test.